pages/confirm.js

```diff
@@ -6,20 +6,17 @@ import { useRouter } from 'next/router'
 import LoadingSpinner from '../components/LoadingSpinner'
 import ErrorDisplay from '../components/ErrorDisplay'
 import api from '../lib/api-client'
+import useLoginRedirect from '../hooks/useLoginRedirect'
 
 const Confirm = () => {
   const [error, setError] = useState(null)
   const router = useRouter()
+  const { user, accessToken } = useLoginRedirect()
```

Comment:
Hooks are executed in order from top to bottom so `useLoginRedirect` should always be the first line of the function.